docs/process/pull-requests.md

```diff
@@ -5,10 +5,10 @@ a consistent process for contributions from the core team and the community.
 
 ## The Review Process
 
-1. **Contributor** opens pull request.
+1. **Contributor** opens a pull request.
 1. When it's ready for review, add a comment to the pull request.
 1. A member of the reviewer team will give it a quick look over and
-   add the `ready-for-review` label and add to [PR Priority Project board](https://github.com/desktop/desktop/projects/21) and any relevant release board.
+   add the `ready-for-review` label and add to the [Community Project board](https://github.com/desktop/desktop/projects/21) and any relevant release board.
```

Comment:
Not sure if https://github.com/desktop/desktop/projects/21 is the appropriate board since it doesn't have any column for community reviews (the current linked project board is closed though).

Reply:
hmmm yeah, i don't think we have a solid process for this anymore. i think right now our informal process is just to add the label and not use a project board.

Reply:
Ok, gonna remove the text that mentions adding the PR to a board! 👍